package.json

```diff
@@ -10,7 +10,8 @@
   "scripts": {
     "build": "webpack --config webpack.config.js --mode development --watch",
     "start": "nodemon server/server.js",
```

Comment:
Similar to 'build' I would reccommend having a build-dev which uses nodemon and a build which uses node

Reply:
Gotcha!  Thanks Ivan : )